AutoMaxLair.py

```diff
@@ -290,7 +290,7 @@ def battle(inst) -> str:
                 f'(index {best_move_index})', 'DEBUG'
             )
             inst.move_index %= 4  # Loop index back to zero if it exceeds 3
-            for __ in range((best_move_index - inst.move_index + 4) % 4):
+            for ___ in range((best_move_index - inst.move_index + 4) % 4):
```

Comment:
Why 3 underscores? I thought 2 was convention.

Reply:
Oh, I thought it was random number of them. It's because __ was already used for a string variable so it was issuing a warning. Let me search for tips online